.github/workflows/zip.yml

```diff
@@ -21,6 +21,29 @@ on:
         default: 'https://github.com/firebase/SpecsStaging.git'
 
 jobs:
+  package-release:
```

Comment:
Add a 2nd package builder that builds with only staged and trunk pods.

The other package build was building from head.